lib/suspenders/generators/app_generator.rb

```diff
@@ -239,7 +239,12 @@ def remove_routes_comment_lines
 
     def outro
       say 'Congratulations! You just pulled our suspenders.'
-      say "Remember to run 'rails generate airbrake' with your API key."
+      say "Remember to run 'bundle exec honeybadger install' with your API key."
```

Comment:
You're right that doesn't read well. `Remember to run 'bundle exec honeybadger install' with your API key.` should probably be an else condition to `if options[:heroku]` instead or maybe `if options[:heroku]` could print something like "This isn't necessary if you're using the Heroku Honeybadger add-on"?

Which do you think is better?